src/scheme/dune

```diff
@@ -0,0 +1,3 @@
+(library
```

Comment:
Missing synopsis about "internal dune lib" and bisect ppx instruementation.

Reply:
synopsis is only needed for public libraries